lib/utils/modify-config-helper.js

```diff
@@ -15,23 +17,58 @@ const runTransform = require("../init/transformations/index");
  * @returns {Function} runTransform - Returns a transformation instance
  */
 
-module.exports = function modifyHelperUtil(action, generator) {
+module.exports = function modifyHelperUtil(action, generator, packages) {
 	let configPath = path.resolve(process.cwd(), "webpack.config.js");
 	const webpackConfigExists = fs.existsSync(configPath);
 	if (!webpackConfigExists) {
 		configPath = null;
 	}
 	const env = yeoman.createEnv("webpack", null);
 	const generatorName = `webpack-${action}-generator`;
+
+	if (!generator) {
+		generator = class extends Generator {
+			initializing() {
+				packages.forEach(pkgPath => {
+					return this.composeWith(require.resolve(pkgPath));
+				});
+			}
+		};
+	}
 	env.registerStub(generator, generatorName);
 
 	env.run(generatorName).on("end", () => {
+		let configModule;
+		try {
+			const configPath = path.resolve(process.cwd(), ".yo-rc.json");
+			configModule = require(configPath);
+			// Change structure of the config to be transformed
+			let tmpConfig = {};
+			Object.keys(configModule).forEach(prop => {
+				const Configs = Object.keys(configModule[prop].configuration);
+				Configs.forEach(config => {
+					tmpConfig[config] = configModule[prop].configuration[config];
+				});
+			});
+			configModule = tmpConfig;
+		} catch (err) {
+			console.error(
+				chalk.red("\nCould not find a yeoman configuration file.\n")
+			);
+			console.error(
+				chalk.red(
+					"\nPlease make sure to use 'this.config.set('configuration', this.configuration);' at the end of the generator.\n"
+				)
+			);
+			Error.stackTraceLimit = 0;
```

Comment:
Doesn't it will set the global error's stackTraceLimit.  I would say don't change it or swallow the error here rather than setting something like this.

Reply:
It's more intuitive for the user. Run the init command with a yeoman generator not using `this.config.set()` (i.e webpack-addons-demo ), and look at the error provided when Stacktrace limit is 0.

Reply:
I would rather not have a package setting this global configuration.

Reply:
They're not using other tools, `webpack init` is a one-timer